variables.tf

```diff
@@ -210,6 +211,13 @@ variable "vmware_vmm_domains" {
     ])
     error_message = "`resolution_immediacy`: Allowed values are `immediate`, `lazy` or `pre-provision`."
   }
+
+  validation {
+    condition = alltrue([
+      for dom in var.vmware_vmm_domains : dom.custom_epg_name == null || try(can(regex("^[a-zA-Z0-9_.-]{0,64}$", dom.custom_epg_name)))
```

Comment:
According to the documentation this can be a string with max. 80 characters: `^.{0,80}$`

Reply:
Should we take regex from description then? ```^[a-zA-Z0-9\\!#$%()*,-./:;@ _{|}~?&+]{0,80}$```

Reply:
No, in this case it is more permissive as every character should be allowed.